[IMP] helpdesk: update sla doc (#7552)
odoo/documentation: content/applications/services/helpdesk/overview/sla.rst

Using the Pivot view
--------------------

By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
Agreement)` policies in the database with tickets that are in progress, failed to fulfill a policy,
or have satisfied a policy are listed. By default, they are grouped by team, and ticket count.

.. image:: sla/sla-status-analysis.png
   :align: center
   :alt: View of the SLA status analysis report in Odoo Helpdesk.

To change the display, or to add additional measurements, click the :guilabel:`Measures` button to
reveal a drop-down menu of reporting criteria, and choose from the options available. Whenever a
measurement is picked, a check mark will appear in the drop-down menu to indicate that that
measurement is included, and a corresponding new column will emerge in the pivot table to show the
relevant calculations.

.. image:: sla/sla-pivot-measures.png
   :align: center
   :alt: View of the available measures in the SLA status analysis report.

To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to :guilabel:`Total`,
and then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon and

> deselect.

Reviewer: I don't see a + next to the total, it shows me a -. I do have a + next to '2 days to start', etc. Maybe just mention in generics that you click a + to add, and a - to remove? This is what I see:
![image](https://github.com/odoo/documentation/assets/121518652/bca3b4ff-a54c-4dc4-a440-5b4704d45cfe)

- :guilabel:`Priority`: the priority level for a ticket is identified by selecting one or more of
  the :guilabel:`⭐ (star)` icons representing the priority level on the kanban card or the ticket
  itself. The :abbr:`SLA (Service Level Agreement)` will **only** be applied after the priority
  level has been updated on the ticket to match the :abbr:`SLA (Service Level Agreement)` criteria.
  If no selection is made in this field, this policy will only apply to tickets marked as `Low
  Priority`, meaning those with zero stars.
- :guilabel:`Tags`: tags are used to indicate what the ticket is about. Multiple tags can be applied
  to a single ticket.
- :guilabel:`Customers`: individual contacts or companies may be selected in this field.
- :guilabel:`Sales Order Items`: this field is available only if a team has the *Timesheets* app
  enabled. This allows the ticket to link directly to a specific line on a sales order, which must
  be indicated on the ticket in the :guilabel:`Sales Order Item` field.

.. note::
   Unless otherwise indicated, multiple selections can be made for each field. For example, multiple

>    :guilabel:`Tags` can be included in a policy, but only one :guilabel:`Priority` level.

Reviewer 2: so the priority is the only field or are there others?

Status Analysis`.

Using the Pivot view
--------------------

By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
Agreement)` policies in the database with tickets that failed to fulfill a policy, are in progress,
or have satisfied a policy are listed. By default, they are grouped by team, and ticket count.

.. image:: sla/sla-status-analysis.png
   :align: center
   :alt: View of the SLA status analysis report in Odoo Helpdesk.

To change the display, or to add additional measurements, click the :guilabel:`Measures` button to
reveal a drop-down menu of reporting criteria, and choose from the options available. Whenever a
measurement is picked, a check mark appears in the drop-down menu to indicate that the measurement
is included, and a corresponding new column will emerge in the pivot table to show the relevant
calculations.

.. image:: sla/sla-pivot-measures.png
   :align: center
   :alt: View of the available measures in the SLA status analysis report.

To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to the policy name and

> then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon and deselect.

Reviewer 2: minus icon on the helpdesk team?

top of the team's settings page, then click :guilabel:`New`.

On the blank :abbr:`SLA (Service Level Agreement)` policy record, enter a :guilabel:`Title` and a
:guilabel:`Description` for the new policy, and proceed to fill out the form using the steps below.

Define the criteria for an SLA policy
-------------------------------------

The :guilabel:`Criteria` section is used to identify which tickets this policy is applied to.

Fill out the following fields to adjust the selection criteria:

.. note::
   Unless otherwise indicated, multiple selections can be made for each field.

- :guilabel:`Team`: a policy can only be applied to one team. *This field is required.*
- :guilabel:`Priority`: the priority level for a ticket is identified by selecting one or more of
  the :guilabel:`⭐ (star)` icons representing the priority level on the kanban card or the ticket
  itself. The :abbr:`SLA (Service Level Agreement)` will **only** be applied after the priority
  level has been updated on the ticket to match the :abbr:`SLA (Service Level Agreement)` criteria.
  If no selection is made in this field, this policy will only apply to tickets marked as `Low

>   Priority`, meaning those with zero stars.

Author: `Type` was removed as a field/ticket identifier in 17.1. Essentially, adding a ticket type was another way to differentiate between tickets, but it operated almost the exact same way as `tags`, and it was deemed unnecessary. I didn't include it to keep the doc as up-to-date as possible and also because it doesn't have an impact on any functionality.

   earliest occurring deadline will be displayed on the ticket. After that deadline has passed, the
   next deadline will be displayed.

.. image:: sla/sla-deadline.png
   :align: center
   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk.

If the :abbr:`SLA (Service Level Agreement)` deadline passes and the ticket has not moved to the
:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag will turn red. After the
:abbr:`SLA (Service Level Agreement)` has failed, the red tag will stay on the ticket, even after
the ticket is moved to the :guilabel:`Reach Stage`.

.. image:: sla/sla-passing-failing.png
   :align: center
   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk.

Analyzing SLA performance
=========================

The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
Agreement)` is fulfilled, as well as the performance of individual team members. Navigate to the
report and corresponding pivot table by going to :menuselection:`Helpdesk app --> Reporting --> SLA
Status Analysis`.

Using the Pivot view
--------------------

> By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level

Reviewer 3: using GUI for the Pivot view in this case seems admissible because that's the name of the button when the cursor hovers over it, unless you think otherwise...

or have satisfied a policy are listed. By default, they are grouped by team and ticket count.

.. image:: sla/sla-status-analysis.png
   :align: center
   :alt: View of the SLA status analysis report in Odoo Helpdesk.

To change the display, or to add additional measurements, click the :guilabel:`Measures` button to
reveal a drop-down menu of reporting criteria, and choose from the options available. Whenever a
measurement is picked, a check mark appears in the drop-down menu to indicate that the measurement
is included, and a corresponding new column will emerge in the pivot table to show the relevant
calculations.

.. image:: sla/sla-pivot-measures.png
   :align: center
   :alt: View of the available measures in the SLA status analysis report.

To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to the policy name and
then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon next to the
policy name.

.. image:: sla/sla-pivot-groups.png
   :align: center
   :alt: View of the available group by options in the SLA status analysis report.

Using the Graph view
--------------------

> The :guilabel:`SLA Status Analysis` report can also be viewed as a :guilabel:`Bar`,
> :guilabel:`Line`, or :guilabel:`Pie` Chart. Toggle between these views by first selecting the
> :guilabel:`Graph` icon at the top-right of the dashboard. Then select the appropriate chart icon at
> the top-left of the graph.

Reviewer 3: ```suggestion
The :guilabel:`SLA Status Analysis` report can also be viewed as a :guilabel:`Bar Chart`,
:guilabel:`Line Chart`, or :guilabel:`Pie Chart`. Toggle between these views by first selecting the
:guilabel:`Graph` button at the top-right of the dashboard. Then, select the appropriate chart icon
at the top-left of the graph.
```

Reviewer 3: added the chart to the end of those graph options because that's the name of the button when the cursor hovers over it. I applied that same methodology to the Graph button, as well.

.. tabs::

   .. tab:: Bar Chart

       .. image:: sla/sla-report-bar.png
          :align: center
          :alt: View of the SLA status analysis report in bar view.

   .. tab:: Line Chart

       .. image:: sla/sla-report-line.png
          :align: center
          :alt: View of the SLA status analysis report in line view.

   .. tab:: Pie Chart

       .. image:: sla/sla-report-pie.png
          :align: center
          :alt: View of the SLA status analysis report in pie chart view.

.. tip::

>    Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` can be viewed :guilabel:`Stacked` by
>    selecting the :guilabel:`Stacked` icon. This presents two or more groups to appear on top of each
>    other instead of next to each other, making it easier to compare data.

Reviewer 3: ```suggestion
   Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` views can be :guilabel:`Stacked` by
   selecting the :guilabel:`Stacked` icon. This displays two or more groups on top of each other,
   instead of next to each other, making it easier to compare data.
```

   .. image:: sla/sla-report-stacked.png
      :align: center
      :alt: View of the SLA status analysis report in bar view, stacked.

> Using the Cohort view
> ---------------------

Reviewer 3: ```suggestion
Cohort view
-----------
```

The :guilabel:`Cohort` view is used to track the changes in data over a period of time. To display

> the :guilabel:`SLA Status Analysis` report in a :guilabel:`Cohort` view, click the icon in the top
> right corner above the chart.

Reviewer 3: ```suggestion
the :guilabel:`SLA Status Analysis` report in a :guilabel:`Cohort` view, click the
:guilabel:`Cohort` button, represented by :guilabel:`four cascading horizontal lines`, in the
top-right corner, next to the other view options.
```

- :guilabel:`Tags`: tags are used to indicate what the ticket is about. Multiple tags can be applied
  to a single ticket.
- :guilabel:`Customers`: individual contacts or companies may be selected in this field.
- :guilabel:`Sales Order Items`: this field is available only if a team has the *Timesheets* app
  enabled. This allows the ticket to link directly to a specific line on a sales order, which must
  be indicated on the ticket in the :guilabel:`Sales Order Items` field.

.. example::
   A support team needs to address urgent issues for VIP customers within one business day.

   The new policy, titled `8 Hours to close`, is assigned to the `VIP Support` team. It **only**
   applies to tickets that are assigned three :guilabel:`⭐ (star)` icons, which equates to an
   `Urgent` priority level.

   At the same time, the tickets can be related to multiple issues, so the policy applies to tickets
   with `Repair`, `Service`, or `Emergency` tags.

   .. image:: sla/sla-create-new.png
      :align: center
      :alt: View of a new SLA policy record with all the relevant information entered.

Establish a target for an SLA policy
------------------------------------

> A *Target* is the stage a ticket needs to reach, and the time allotted to reach that stage, in order

Reviewer 4: Not a proper noun

```suggestion
A *target* is the stage a ticket needs to reach, and the time allotted to reach that stage, in order
```

  `Low Priority`, meaning those with zero :guilabel:`⭐ (star)` icons.
- :guilabel:`Tags`: tags are used to indicate what the ticket is about. Multiple tags can be applied
  to a single ticket.
- :guilabel:`Customers`: individual contacts or companies may be selected in this field.
- :guilabel:`Sales Order Items`: this field is available only if a team has the *Timesheets* app
  enabled. This allows the ticket to link directly to a specific line on a sales order, which must
  be indicated on the ticket in the :guilabel:`Sales Order Items` field.

.. example::
   A support team needs to address urgent issues for VIP customers within one business day.

   The new policy, titled `8 Hours to close`, is assigned to the `VIP Support` team. It **only**
   applies to tickets that are assigned three :guilabel:`⭐ (star)` icons, which equates to an
   `Urgent` priority level.

   At the same time, the tickets can be related to multiple issues, so the policy applies to tickets
   with `Repair`, `Service`, or `Emergency` tags.

   .. image:: sla/sla-create-new.png
      :align: center
      :alt: View of a new SLA policy record with all the relevant information entered.

> 

Reviewer 5: Remove extra line
```suggestion
```

Level Agreement)` tag turns green, and the deadline disappears from view on the ticket.

.. image:: sla/sla-deadline.png
   :align: center
   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk.

.. important::
   If a ticket fits the criteria for more than one :abbr:`SLA (Service Level Agreement)`, the
   earliest occurring deadline is displayed on the ticket. After that deadline has passed, the next
   deadline is displayed.

If the :abbr:`SLA (Service Level Agreement)` deadline passes and the ticket has not moved to the
:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag turns red. After the
:abbr:`SLA (Service Level Agreement)` has failed, the red tag stays on the ticket, even after the
ticket is moved to the :guilabel:`Reach Stage`.

.. image:: sla/sla-passing-failing.png
   :align: center
   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk.

Analyze SLA performance
=======================

The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
Agreement)` is fulfilled, as well as the performance of individual team members. Navigate to the

> report, and corresponding Pivot table, by going to :menuselection:`Helpdesk app --> Reporting -->

Reviewer 5: ```suggestion
report, and corresponding pivot table, by going to :menuselection:`Helpdesk app --> Reporting -->
```

SLA Status Analysis`.

Pivot view
----------

By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
Agreement)` policies in the database with tickets that failed to fulfill a policy, are in progress,
or have satisfied a policy are listed. By default, they are grouped by team and ticket count.

.. image:: sla/sla-status-analysis.png
   :align: center
   :alt: View of the SLA status analysis report in Odoo Helpdesk.

To change the display, or add additional measurements, click the :guilabel:`Measures` button to
reveal a drop-down menu of reporting criteria, and choose from the options available.

Whenever a measurement is picked, a :guilabel:`✔️ (checkmark)` icon appears in the drop-down menu to

> indicate that the measurement is included, and a corresponding new column emerges in the Pivot table

Reviewer 5: The term _pivot table_ generic, no need to capitalize
```suggestion
indicate that the measurement is included, and a corresponding new column emerges in the pivot table
```

at the top-left of the graph.

.. tabs::

   .. tab:: Bar Chart

       .. image:: sla/sla-report-bar.png
          :align: center
          :alt: View of the SLA status analysis report in bar view.

   .. tab:: Line Chart

       .. image:: sla/sla-report-line.png
          :align: center
          :alt: View of the SLA status analysis report in line view.

   .. tab:: Pie Chart

       .. image:: sla/sla-report-pie.png
          :align: center
          :alt: View of the SLA status analysis report in pie chart view.

.. tip::
   Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` views can be :guilabel:`Stacked` by
   selecting the :guilabel:`Stacked` icon. This displays two or more groups on top of each other
   instead of next to each other, making it easier to compare data.

   .. image:: sla/sla-report-stacked.png
      :align: center
      :alt: View of the SLA status analysis report in bar view, stacked.

Cohort view
-----------

The :guilabel:`Cohort` view is used to track the changes in data over a period of time. To display
the :guilabel:`SLA Status Analysis` report in a :guilabel:`Cohort` view, click the

> :guilabel:`Cohort` button, represented by :guilabel:`four cascading horizontal lines`, in the

Reviewer 5: Add parentheses around an icon descriptor for consistency 
```suggestion
:guilabel:`Cohort` button, represented by :guilabel:`(four cascading horizontal lines)`, in the
```

       .. image:: sla/sla-report-bar.png
          :align: center

>           :alt: View of the SLA status analysis report in bar view.

Reviewer: I can't make a comment in line, but for line 150, I would say 'SLA Status Analysis` report since SLA is in the name in the menu when you click on it,

Reviewer: Also, in line 152, I would mention the buttons to change the view are in the top right of the chart, then they appear above the chart. I couldn't not find them at first and didn't realize they were tucked all the way in the corner.

   .. tab:: Line Chart

       .. image:: sla/sla-report-line.png
          :align: center

>           :alt: View of the SLA status analysis report in line view.

Reviewer: i LOVE that the graphs are in tabs! Great idea!!!

.. image:: sla/sla-report-cohort.png
   :align: center

>    :alt: View of the SLA status analysis report in cohort view.

Reviewer: For line 183, I'd add SLA before the 'Status Analysis' since that's the full name.
